db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_11_02_120213) do
+ActiveRecord::Schema.define(version: 2020_11_02_160605) do
```

Comment:
This feels like it shouldn't affect the schema 🤔 .

Reply:
Happy to be taught otherwise :)

Reply:
The only way around this that I can think of is to have it as a rake task and manually run it, but I'm happy to be told otherwise also :D

Reply:
Migration has now been replaced with a rake task, which can be used as so:

```
rake paper_trail:populate_with_suppliers
rake paper_trail:populate_with_suppliers\[up\]
rake paper_trail:populate_with_suppliers\[down\]
```

If no argument is passed, it will default to `up`.